shub/schedule.py

```diff
@@ -0,0 +1,41 @@
+from ConfigParser import NoSectionError, NoOptionError
+import click
+from click import ClickException
+from scrapinghub import Connection
+from shub import scrapycfg
+from shub.utils import find_api_key
+
+
+@click.command(help='Schedule a spider to run on Scrapy Cloud')
+@click.pass_context
+@click.argument("spider", type=click.STRING)
+@click.option("-p", "--project-id", help="the project ID", type=click.INT)
+@click.option("-a", "--argument", help="argument for the spider (-a name=value)", multiple=True)
+def cli(context, project_id, spider, argument):
+    apikey = find_api_key()
+    if not apikey:
+        raise ClickException('Scrapinghub API key not found: please, run \'scrapy login\' first')
+    project_id = project_id or get_project_id_from_config()
+    job_key = schedule_spider(apikey, project_id, spider, argument)
+    click.echo(
+        'Spider {} scheduled: {}\nCheck the spider execution at: '
+        'https://dash.scrapinghub.com/p/{}/job/{}/{}'.format(spider, job_key, *job_key.split('/'))
+    )
+
+
+def schedule_spider(apikey, project_id, spider, arguments=()):
```

Comment:
I believe the point of raising ClickException here is sending a nicer message to user.

The messages can be improved -- instead of `"doesn't exist"`, it's better `"project {} not found"` (it can be that the project exists but the used credentials don't have access to it).